packetbeat/protos/tls/_meta/fields.yml

```diff
@@ -1,410 +1,436 @@
-- key: tls
-  title: "TLS"
+- key: tls_detailed
+  title: "Detailed TLS"
   description: >
-    TLS-specific event fields.
+    Detailed TLS-specific event fields.
   fields:
     - name: tls
       type: group
       fields:
-        - name: version
-          type: keyword
-          description: >
-            The version of the TLS protocol used.
-          example: "TLS 1.3"
-
-        - name: handshake_completed
-          type: boolean
-          description: >
-            Whether the TLS negotiation has been successful and the session has
-            transitioned to encrypted mode.
-
-        - name: resumed
-          type: boolean
-          description: >
-            If the TLS session has been resumed from a previous session.
-
-        - name: resumption_method
-          type: keyword
-          description: >
-            If the session has been resumed, the underlying method used. One of
-            "id" for TLS session ID or "ticket" for TLS ticket extension.
-
-        - name: client_certificate_requested
-          type: boolean
-          description: >
-            Whether the server has requested the client to authenticate itself
-            using a client certificate.
-
-        - name: client_hello
+        - name: detailed
```

Comment:
Moving everything that's not ECS under `tls.detailed.*` will be clearer indeed. But isn't that a breaking change?